activesupport/lib/active_support/evented_file_update_checker.rb

```diff
@@ -107,11 +105,11 @@ def execute_if_updated
 
     private
       def boot!
-        Listen.to(*@dtw, &method(:changed)).start
+        ::Listen.to(*@dtw, &method(:changed)).start
```

Comment:
There is no reason to use the `::` here.

Reply:
IMO this error:
```
uninitialized constant Listen (NameError)
```
speaks louder than this one:
```
uninitialized constant ActiveSupport::EventedFileUpdateChecker::Listen (NameError)
```
ruby constant resolution adds noise to the real issue. But if it makes more sense to you to keep it like before then ok.

Reply:
Good point. But if we move the require to the top of the file we will not have this execption anymore.

Reply:
Done!